source/manual/fall-back-to-mirror.html.md

```diff
@@ -12,6 +12,45 @@ A GOV.UK mirror is a static copy of pages and assets hosted on www.gov.uk or ass
 - related assets for those pages (e.g. JavaScript, CSS, images, fonts)
 - other linked assets (or "attachments") such as CSVs, PDFs etc
 
+## Architecture
+
+### How content flows through the mirror system
+
+The GOV.UK mirror system operates in two main flows:
+
+#### 1. Content Population Flow (Nightly)
+
+The [govuk-mirror-sync cronjob][govuk-mirror-sync configuration] crawls content, saves it locally, and distributes it across multiple storage locations for redundancy.
+
+```mermaid
+flowchart TB
+    A[govuk-mirror-sync<br/>cronjob in Argo] --> B[Crawl www &<br/>assets domains]
+    B --> C[Save pages<br/>to local disk]
```

Comment:
Needs an extra step to explain that the same batch job copies the from local disk to the primary mirror bucket